doctrinal-works-v1/original/11-chapter-9.adoc

```diff
@@ -18,7 +18,7 @@ fulness and gluttony, and serving the creature more than the Creator,
 who is God blessed forever; and serving your lusts more than Christ,
 and making your bellies your gods, fatting up yourselves with the creature,
 until you have forgotten God, until you grow proud and wanton:
-alas! heap not up treasure upon earth, where thieves, and rust, and canker is,
+alas! Heap not up treasure upon earth, where thieves, and rust, and canker is,
```

Comment:
This is no problem, and I'm going to leave it, but going forward, if the edition is _original_ and you see uncapitalized words after a `!` (or sometimes a `?`), you can just leave them.  That used to be grammatically correct, and in original editions, we generally leave them as is.  In modernized versions we update the capitalization, sometimes adding a `--` if necessary, but you can ignore them in _original_ editions. 👍